Improve API Docs (#98)
ballerina-platform/module-ballerina-regex: regex-ballerina/Module.md

> ## Module Overview

Reviewer: ```suggestion
## Overview
```

This module provides functions for searching, splitting, and replacing the set of characters of the string. These APIs are using a 

> regular expression as a `String` to perform these operations by finding matches string.

Reviewer 2: ```suggestion
regular expression as a `String` to perform these operations by finding the string matches.
```

> [Regular expressions](https://en.wikipedia.org/wiki/Regular_expression), are a notation for describing sets of 

Reviewer 3: ```suggestion
[Regular expressions](https://en.wikipedia.org/wiki/Regular_expression), are notations for describing sets of 
```
